sdks/python/apache_beam/transforms/core.py

```diff
@@ -915,10 +914,10 @@ class CombinePerKey(PTransformWithSideInputs):
   Args:
     pcoll: input pcollection.
     fn: instance of CombineFn to apply to all values under the same key in
-      pcoll, or a callable whose signature is f(iterable, *args, **kwargs)
+      pcoll, or a callable whose signature is f(iterable, \*args, \**kwargs)
       (e.g., sum, max).
-    *args: arguments and side inputs, passed directly to the CombineFn.
-    **kwargs: arguments and side inputs, passed directly to the CombineFn.
+    \*args: arguments and side inputs, passed directly to the CombineFn.
```

Comment:
Thanks for catching. It was only needed in the line above this and not here. Fixed now.